### PR TITLE
Add a simpler example to README

### PR DESCRIPTION
**Pull request by kira-lin**
I add a word count example, which can run without modification and any other dependency. People can try this right away after they install raydp. Besides, I add some text to help understand the purpose of raydp.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,28 @@ pip install dist/raydp*.whl
...
+import ray
+import raydp
+
+ray.init(address='auto')
+
+spark = raydp.init_spark('word_count',
+                         num_executors=2,
+                         executor_cores=2,
+                         executor_memory='1GB')
+wordsDF = spark.createDataFrame([('look',), ('spark',), ('tutorial',),
```
Comment (ConeyLiu) [inline README.md:47]: replace `wordsDF` to `df` ?

File: README.md [hunk 2]
```diff
@@ -32,6 +32,28 @@ pip install dist/raydp*.whl
...
+ray.init(address='auto')
+
+spark = raydp.init_spark('word_count',
+                         num_executors=2,
+                         executor_cores=2,
+                         executor_memory='1GB')
+wordsDF = spark.createDataFrame([('look',), ('spark',), ('tutorial',), 
+                                ('spark',), ('look', ), ('python', )], ['word'])
+wordsDF.show()
+wordCountsDF = (wordsDF
```
Comment (ConeyLiu) [inline README.md:50]: single line should be enough

File: README.md [hunk 3]
```diff
@@ -32,6 +32,28 @@ pip install dist/raydp*.whl
...
+                .groupBy('word').count())
+wordCountsDF.show()
```
Comment (ConeyLiu) [inline README.md:52]: add `raydp.stop_spark(..)`

Comment (ConeyLiu) [reply README.md:52]: Could you add the `raydp.stop_spark(...)` in the later example as well?

**Comment by ConeyLiu** [comment]
Thanks @kira-lin, others LGTM.

File: README.md (hunk 1 above)
Comment (kira-lin) [reply README.md:47]: ok

File: README.md (hunk 2 above)
Comment (kira-lin) [reply README.md:50]: ok

File: README.md (hunk 3 above)
Comment (kira-lin) [reply README.md:52]: sure

**Pull request merged**